app-admin/doctl/doctl-1.54.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 go-module
+
+DESCRIPTION="A command line tool for DigitalOcean services"
+HOMEPAGE="https://github.com/digitalocean/doctl"
+SRC_URI="https://github.com/digitalocean/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
```

Comment:
Sorry that I failed to mention this the first time, but these packages need to include licenses for bundled go modules as well. There are tools in the ::gentoo repo to check these, but I can't say which is the best one. See go-licenses, golicense and go-license-detector.

Reply:
I know about these tools and use golicense in new ebuilds, this one is relatively old and I didn't run golicense against it.
But what I didn't know that mentioning all licenses in ebuild is mandatory. Will take this into account, thanks.
License for go modules added.